stubs/ext-random.phpstub

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * PHP 8.2 introduces a new PHP extension named "random".
+ * @see https://github.com/php/php-src/blob/master/ext/random/random.stub.php
+ * @see https://php.watch/versions/8.2/ext-random
+ */
+
+define('MT_RAND_MT19937', 0);
+
+define('MT_RAND_PHP', 1);
+
+namespace {
+    function lcg_value(): float {}
+
+    function mt_srand(int $seed = 0, int $mode = MT_RAND_MT19937): void {}
+
+    function srand(int $seed = 0, int $mode = MT_RAND_MT19937): void {}
+
+    function rand(int $min, int $max): int {}
+
+    function mt_rand(int $min, int $max): int {}
+
+    function mt_getrandmax(): int {}
+
+    function getrandmax(): int {}
+
+    function random_bytes(int $length): string {}
+
+    function random_int(int $min, int $max): int {}
+}
+
+namespace Random\Engine
+{
+    final class Mt19937 implements \Random\Engine
+    {
+        public function __construct(int|null $seed = null, int $mode = MT_RAND_MT19937) {}
+
+        public function generate(): string {}
+
+        public function __serialize(): array {}
+
+        public function __unserialize(array $data): void {}
+
+        public function __debugInfo(): array {}
+    }
+
+    final class PcgOneseq128XslRr64 implements \Random\Engine
+    {
+        public function __construct(string|int|null $seed = null) {}
+
+        public function generate(): string {}
+
+        public function jump(int $advance): void {}
+
+        public function __serialize(): array {}
+
+        public function __unserialize(array $data): void {}
+
+        public function __debugInfo(): array {}
+    }
+
+    final class Xoshiro256StarStar implements \Random\Engine
+    {
+        public function __construct(string|int|null $seed = null) {}
+
+        public function generate(): string {}
+
+        public function jump(): void {}
+
+        public function jumpLong(): void {}
+
+        public function __serialize(): array {}
+
+        public function __unserialize(array $data): void {}
+
+        public function __debugInfo(): array {}
+    }
+
+    final class Secure implements \Random\CryptoSafeEngine
+    {
+        public function generate(): string {}
+    }
+}
+
+namespace Random
+{
+    interface Engine
+    {
+        public function generate(): string;
+    }
+
+    interface CryptoSafeEngine extends Engine
+    {
+    }
+
+    final class Randomizer
+    {
+        public readonly Engine $engine;
+
+        public function __construct(?Engine $engine = null) {}
+
+        public function nextInt(): int {}
+
+        public function getInt(int $min, int $max): int {}
+
+        public function getBytes(int $length): string {}
+
+        public function shuffleArray(array $array): array {}
```

Comment:
I would like to add more specific array info later as a separate PR (if it's possible)

Reply:
Fine by me :)